markdown/GameGuides/RoboCopRogueCity.md

```diff
@@ -0,0 +1,58 @@
+![Hogwarts Legacy](Images\robocop_rogue_city_header.png "Shot by Dogmod"){.shadowed .autosize}
+## Summary
+Feature | Supported
+--|--
+Vanilla Photo Mode | No
+Hotsampling | Yes (use windowed fullscreen)
+DSR | No
```

Comment:
I'm not sure how to check for DSR more extensively. Nvidia Control Panel and usual methods i managed to find didn't work on the game.

Reply:
If you have DSR enabled in the nvidia control panel, and you can't pick a resolution higher than your monitor's resolution then it won't work. Usually tho, DSR works fine in UE games.

Reply:
Try adding resolutions using the [Custom DSR resolutions guide](https://framedsc.com/GeneralGuides/custom_dsr_resolutions.htm) and see if you can select them now (you gotta play in fullscreen I believe? not sure). It might seem bothersome at first but it's worth the hassle when it comes to writing guides.

Reply:
I've checked it again and can confirm that DSR is working.